src/Interfaces/AbstractPolynomialZonotope.jl

```diff
@@ -0,0 +1,18 @@
+"""
+    AbstractPolynomialZonotope{N} <: LazySet{N}
+
+Abstract type for polynomial zonotope sets.
+
+### Notes
+
+Polynomial zonotopes are in general non-convex. They are always bounded.
+
+```jldoctest; setup = :(using LazySets: subtypes)
+julia> subtypes(AbstractPolynomialZonotope)
+1-element Vector{Any}:
+ DensePolynomialZonotope
+```
+"""
+abstract type AbstractPolynomialZonotope{N} end
+
+isconvextype(::Type{<:AbstractPolynomialZonotope}) = false
```

Comment:
To match the documentation:
```suggestion
isconvextype(::Type{<:AbstractPolynomialZonotope}) = false
isboundedtype(::Type{<:AbstractPolynomialZonotope}) = true
```